pingcap/tidb
# metrics: add metrics for multi-statement

### What problem does this PR solve?


Issue Number: close #38930

Problem Summary:

### What is changed and how it works?
Add metrics for it.

### Check List

Tests 


1. start a cluster
2. run some multi-statement queries
3. check out this metrics in Grafana
<img width="821" alt="image" src="https://user-images.githubusercontent.com/4352397/200240593-c924dc78-a713-4562-b46a-383d2330cdd2.png">


Side effects


Documentation


### Release note



Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```
